ksqldb-streams/src/test/java/io/confluent/ksql/execution/streams/SourceBuilderTest.java

```diff
@@ -464,6 +469,41 @@ public void shouldUseCorrectSerdeForUnwindowedKey() {
     );
   }
 
+  @Test
+  public void shouldNotContainSchemaIdInMaterializedSchema() {
+    when(keyFormatInfo.copyWithoutProperty("schemaId")).thenReturn(keyFormatInfoWithoutProps);
+    when(valueFormatInfo.copyWithoutProperty("schemaId")).thenReturn(valueFormatInfoWithoutProps);
```

Comment:
Can you move these two lines into the `setup()` method? This way the other tests which check for the regular `keyFormatInfo` or `valueFormatInfo` get coverage from the changes in this PR too (i.e., the other tests help verify that the changes in this PR are _only_ applied to materialization and not any of the source consumeds).

Reply:
Sure